From 549935084bb4b29d77513790495885dc66db980f Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Fri, 19 Oct 2018 17:20:16 +0200
Subject: [PATCH] tqma8qx: next fix for default fdt_file

now that we have proper files in linux, fix again.

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 include/configs/tqma8qx.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/tqma8qx.h b/include/configs/tqma8qx.h
index a81688aa83..efff1213f1 100644
--- a/include/configs/tqma8qx.h
+++ b/include/configs/tqma8qx.h
@@ -115,7 +115,7 @@
 	"cntr_addr=0x88000000\0"			\
 	"cntr_file=os_cntr_signed.bin\0" \
 	"boot_fdt=try\0" \
-	"fdt_file=fsl-imx8qxp-tqma8qx-a0.dtb\0" \
+	"fdt_file=fsl-imx8qxp-tqma8qx-a0-mba8qx.dtb\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
 	"mmcpart=" __stringify(CONFIG_SYS_MMC_IMG_LOAD_PART) "\0" \
 	"mmcroot=" CONFIG_MMCROOT " rootwait rw\0" \
